.github/pull_request_template.md

```diff
@@ -0,0 +1,84 @@
+<!--
+You've got a Pull Request you want to submit? Awesome!
+This PR template is here to help ensure you're setup for success:
+  please fill it out to help ensure that your PR is complete and ready for approval.
+-->
+
+**JIRA Ticket:**
+<https://jira.cms.gov/browse/xxxx>
+
+**User Story or Bug Summary:**
+<!-- Please copy-paste the brief user story or bug description that this PR is intended to address. -->
+
+
+### What Does This PR Do?
+
+<!--
+Add detailed discussion of changes here
+This is likely a summary, or the complete contents, of your commit messages.
+-->
+
+
+### What Should Reviewers Watch For?
+
+If you're reviewing this PR, please check these things, in particular:
+
+* TODO
+
+<!--
+Add some items to the list above, or remove the entire section if it doesn't apply for some reason.
+
+Common items include:
+* Is this likely to address the goals expressed in the user story?
+* Are any additional documentation updates needed?
+* Are there any unhandled and/or untested edge cases you can think of?
+* Is user input properly sanitized & handled?
+* Does this make any backwards-incompatible changes that might break end user clients?
+* Can you find any bugs if you run the code locally and test it manually?
+-->
+
+
+### What Security Implications Does This PR Have?
+
+Submitters should complete the following questionnaire:
+
+* If the answer to any of the questions below is **Yes**, then here's a link to the associated Security Impact Assessment: N/A.
+    * Does this PR add any new software dependencies? **Yes** or **No**.
+    * Does this PR modify or invalidate any of our security controls? **Yes** or **No**.
+    * Does this PR store or transmit data that was not stored or transmitted before? **Yes** or **No**.
+* If the answer to any of the questions below is **Yes**, then please add @StewGoin as a reviewer, and note that this PR should not be merged unless/until he also approves it.
+    * Do you think this PR requires additional review of its security implications for other reasons? **Yes** or **No**.
+
+
+### What Needs to Be Deployed Before this PR?
+
+This PR cannot be deployed until the following pre-requisite changes have been fully deployed:
+
+* CMSgov/some_repo#42
+
+<!--
+Add some items to the list above, or remove the entire section if it doesn't apply.
+
+Common items include:
+* Database migrations (which should always go out in PRs by themselves, to reduce risk).
+* New features in external dependencies (e.g. BFD).
+-->
+
+
+### Submitter Checklist
+
+Before requesting final review, I've gone through and verified that this PR complies with the following requirements:
+
+* [ ] I've verified that this PR is as small as it can reasonably be, in order to:
```

Comment:
What would you want a developer to do if they realize, "Oh no, this is definitely too big of a PR!"? Is there a standard or best-practice for refining the scope of a PR that we could point to? Basically, how we'd like them to dig themselves out of the hole?

Reply:
Great suggestion!

I had trouble finding articles that were 100% on-topic for this, but I just tweaked the wording and added a couple of references that I think get the job done in concert.

Better?

Reply:
Seconding @johnfrenchxyz 's question. I'm not sure there's a clean way to recover if you've reached the point of creating a large PR. This is something we need to be mindful of when creating tickets and while work is being done. Are you making more changes than you expected? Can this work be broken into smaller tasks?

Reply:
I think your update gives more context. And I agree that this is more of a pre-PR challenge in ticket creation, grooming, etc. I live having the check from a culture standpoint. I think I am good with the checkbox now if we push to make this a priority in ticket creation.

Reply:
FWIW, you can break up a branch/PR after it's gotten too large. It's not a lot of fun, but I've done it.

That said, both of you are **very** correct that it's better to try and prevent than to fix! I'll leave the wording here, as updated, for now but leave a mental note that this might need to be revisited in the future. (Don't think I can add a comment to the Markdown in the middle of the list, or I'd leave a note there for submitters. 🤷)

Reply:
If we're intending prevention rather than fixing, I'm not sure the PR template is the right place for it.  We risk having so much in here that no one reads/thinks through it all.